googleMapsApiModule.py

```diff
@@ -27,3 +27,13 @@ def mapsstatic(search_location):
     result_url = f'https://maps.googleapis.com/maps/api/staticmap?center={address.lower().replace(" ", "+")}&zoom=13&scale=1&size=600x350&maptype=roadmap&key={config.key}&format=png&visual_refresh=true&markers=size:mid%7Ccolor:0xff0000%7Clabel:L%7C{address.lower().replace(" ", "+")}'
     print(result_url)
     webbrowser.open_new(result_url)
+
+def elevation(search_location):
```

Comment:
@adityabisoi Can you add a docstring which describes the purpose of this function and the way to use it? I know we haven't done that for any of the functions yet but that is something I have in mind as an enhancement and this is a good starting point since this is a new function.